bird.cpp

```diff
@@ -39,10 +39,14 @@ void Bird::draw()
 {
 	glDepthMask(GL_FALSE);
 
-	glm::mat4 identity = glm::mat4(1.0f);
-	glm::mat4 result = identity;
+	// Pitch (forward tilt in planespeak) calculation.
+	float pitch = (float) atan(velocity.y)*5000; // TODO Trial-and-error value
```

Comment:
An even better solution would be to use glm's quaternion rotate between 2 vectors (see [here](http://glm.g-truc.net/0.9.5/api/a00222.html#gaac3f65ac2ca7ac7b22bf18d9f6fd4e6e)). 

I'd rather move directly to the proper approach here. I'll put up a PR tomorrow morning to move Bleepy and have the cam follow him.